src/migrator/utils.ts

```diff
@@ -325,7 +325,9 @@ WHERE "migration"."model name" = ${1}`,
 	if (data == null) {
 		return [];
 	}
-	return sbvrUtils.sbvrTypes.JSON.fetchProcessing(data.executed_migrations);
+	return sbvrUtils.sbvrTypes.JSON.fetchProcessing(
+		data.executed_migrations,
+	) as string[];
```

Comment:
Is this cast necessary? Can it be `satisfies` instead?

Reply:
@Page- 
No as it doesn't satisfy. 
Also a type guard isn't working with the current typings and typescript inferencing capabilities. 
Left a comment and leave it as a cast.